package.json

```diff
@@ -6,24 +6,22 @@
     "packages/*"
   ],
   "dependencies": {
-    "husky": "8.0.1",
-    "lerna": "5.5.0"
+    "husky": "8.0.1"
   },
   "devDependencies": {
     "lerna-update-wizard": "1.1.0"
   },
   "scripts": {
-    "build": "lerna run build --ignore datagateway-common",
-    "lint": "lerna run lint:js -- --max-warnings=0",
-    "tsc": "lerna run tsc",
+    "build": "yarn workspaces foreach --verbose --parallel --exclude datagateway-common run build",
+    "lint": "yarn workspaces foreach --verbose --parallel run lint:js --max-warnings=0",
+    "tsc": "yarn workspace datagateway-common tsc",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit": "lerna run test --stream",
-    "test:e2e": "cd packages/datagateway-dataview && yarn run e2e && cd ../datagateway-download && yarn run e2e && cd ../datagateway-search && yarn run e2e",
-    "test:e2e:lerna": "lerna run e2e --stream --concurrency 1",
+    "test:unit": "yarn workspaces foreach --interlaced --verbose --exclude datagateway run test",
```

Comment:
This command seems to run significantly slower than we used to with lerna. Could we add `--parallel` and `--jobs 3` to it?

Reply:
Also, what is the `--exclude` flag doing here?

Reply:
So with the exclude tag, I was having issues with that command as the `foreach` option also was picking up the top level package (rather than just the ones in `packages/*`) so it would basically result in a recursion mess (as it also would run `"test": "yarn test:unit && yarn test:e2e"`). We can either specify which packages to run explicitly, or exclude the datagateway package, and I think excluding is a tidier solution (although I haven't tested which preforms better)

Reply:
I thought it might be that. Your solution is just fine!

Reply:
Okay nice, thanks. And on the performance, I have been performance testing locally mainly and my PC didn't handle most of these commands in parallel well. Obviously this should be fine when running in the CI (as I assume its running on a beefy CPU) but I thought it would be best to optimise around running the commands locally, however it makes more sense to optimise around the CI. Ill look into this, thanks :)

Reply:
Yes, I agree, it's hard to compare the two and hard to tell if I had a good run on my laptop by chance, doesn't necessarily mean it's definitely performing better. I would say try running the commands on the CI and compare the two runs - if one is better than the other, use that. Like you say, I'm personally never running the global commands, they're more for the CI